Source/Evolve/WarpXEvolve.cpp

```diff
@@ -802,8 +795,7 @@ WarpX::OneStep_sub1 (Real curtime)
     if ( safe_guard_cells )
         FillBoundaryB(coarse_lev, PatchType::fine, guard_cells.ng_FieldSolver);
 
-    // Synchronize all nodal points at the end of the timestep
-    NodalSync(Efield_fp, Efield_cp);
+    // Synchronize nodal points at the end of the time step
     NodalSync(Bfield_fp, Bfield_cp);
```

Comment:
The `NodalSync` for the B field and F field should be remove too, no?

Reply:
I thought this was making some CI tests fail, but I removed all calls in https://github.com/ECP-WarpX/WarpX/pull/3013/commits/54d9d776e90c0606810164dbdb48cb0810506096 and we can see how CI performs.